hooks/post_gen_project.py

```diff
@@ -172,11 +172,22 @@ def clone_gh_repo(url):
     """
     pass
 
-def load_headers(project_path):
-    """Find usepackages.txt ,newcommands.txt, and manuscript.tex in
+def load_headers(project_path, manuscript_file_name="manuscript.tex"):
+    r"""Loads user-defined latex packages and new-commands into the
+    mauscript template tex file.
+
+    Find usepackages.txt, newcommands.txt, and manuscript.tex in
     project directory. Insert usepacakgea and new commands into
     manuscript.tex.
 
+    Example content of usepackages.txt:
+    \\usepackage{mathtools}
```

Comment:
how do I compose this file if I want to add two packages?  Do I make a comma separated list? space separated list? What do I do?  Your example, unfortunately, is too simple to capture even common multiple cases.